Makefile

```diff
@@ -28,6 +28,7 @@ build:
 
 tarball: build
 	cd $(TMP) && tar cfj SOURCES/$(PACKAGE).tar.bz2 $(PACKAGE)
+	@printf "$(shell realpath $(TMP)/SOURCES/$(PACKAGE).tar.bz2)\n"
```

Comment:
I think the problem here is that the path is absolute - it should ideally be relative within the project

Reply:
I guess this the same issue as psss/tmt/pull/249. I've tried relative path there as well but it did not help.